docs/conf.py

```diff
@@ -142,6 +141,13 @@
     )
 
 # Theme options for pydata_sphinx_theme
+# We don't check switcher because there are 3 possible states for a repo:
+# 1. New project, docs are not published so there is no switcher
+# 2. Existing project with latest skeleton, switcher should be there
+# 3. Existing project with skeleton being either adopted or updated,
+#    switcher may or mar not be there and its state is unknown.
+# Point 3 makes checking switcher difficult, because if it is not as
+# the theme expects, it needs to be manually fixed on every branch.
```

Comment:
```suggestion
# We don't check switcher because there are 3 possible states for a repo:
# 1. New project, docs are not published so there is no switcher
# 2. Existing project with latest skeleton, switcher exists and works
# 3. Existing project with old skeleton that makes broken switcher,
#    switcher exists but is broken
# Point 3 makes checking switcher difficult, because the updated skeleton
# will fix the switcher at the end of the docs workflow, but never gets a chance
# to complete as the docs build warns and fails.
```